### DISCUSSION TITLE
HTTP and HTTPS Option for Configuration

### DISCUSSION THREAD

**Discussion by shantanuneosoft** [NeoSOFT-Technologies/rest-node-nestjs#32] [Ideas]
The boilerplate should have both HTTP and HTTPS option for the URL

**Comment by suresh-shetiar** [comment]
Can you please elaborate in what case you would like to use HTTP endpoint! and in what way this could be beneficial. Thanks

**Reply by santoshshinde2012** [reply]
I think we can create SSL based script to handle HTTPS. But we have to think about the different headers for both protocols @shantanuneosoft @suresh-shetiar

**Reply by shantanuneosoft** [reply]
This will be helpful if any project required HTTPS endpoint otherwise it will cause to mixed content issue while fetching the data through API. @suresh-shetiar
